source/includes/inventory/introduction.md

```diff
@@ -2,4 +2,12 @@
 
 # Inventory App for retailers
 
-(Redaction in progress)
\ No newline at end of file
+The inventory app is a mobile app (iOS and Android) to process your incoming inventory.
```

Comment:
You must go further, "process" is too generic.

You must explain for who this application is for, what it will change to use it and why it is recommended to use it.